### ISSUE TITLE
Problems with Med Menu and Bindings

### ISSUE THREAD

**Issue by worsin** [acemod/ACE3#5202]
**Arma 3 Version:** `1.70`
**CBA Version:** `3.3.1` 
**ACE3 Version:** `3.9.2`

**Mods:**
```
- CBA_A3
- ace
- Overthrow
- SSPCM
```

**Description:**
The H key will not open up the Medical Menu, Overthrow devs say that the setting to use med menu is enabled. So i try to reassign the H in case there is a conflict somewhere but i cant even do that.

I go in Controls >> Configure Addons. I select the Ace3 addon i want to modify keybinds for.

Down at the bottom is says "Double Click any action to change its binding"

Double Clicking does nothing. I can delete the bind and i can set it to default but i cannot change them.

So basically cannot open med menu and cannot change any bindings.

**Steps to reproduce:**
n/a

**Where did the issue occur?**
Singleplayer

**Placed Modules:**
- Not sure what this means

**RPT log file:**
- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file. An instruction to find your RPT files can be found [here](https://community.bistudio.com/wiki/Crash_Files#Arma_3).
- If possible at the time the bug is encountered, go to ACE Options and select "Debug To Clipboard", this will print extensive debug information to the RPT file.

**Comment by worsin** [comment]
Ok well it turns out that the Med Menu setting WAS disabled.

The bindings area though is still buggy.  I have to spam click the binding like 20 times before it realizes i double clicked and THEN i can change it.

**Comment by PabstMirror** [comment]
Keybinding is handled by CBA
It is getting a full rework in the next version (3.4)

**Comment by PabstMirror** [comment]
Issue is CBA's domain https://github.com/CBATeam/CBA_A3
let them know if you are still having problems after CBA 3.4 is released
